pkg/syncer/metadatasyncer.go

```diff
@@ -1065,6 +1067,14 @@ func csiPVDeleted(ctx context.Context, pv *v1.PersistentVolume, metadataSyncer *
 		if err := metadataSyncer.volumeManager.DeleteVolume(ctx, volumeHandle, deleteDisk); err != nil {
 			log.Errorf("PVDeleted: Failed to delete disk %s with error %+v", volumeHandle, err)
 		}
+		if migrationFeatureEnabled && pv.Spec.VsphereVolume != nil {
+			// Delete the cnsvspherevolumemigration crd instance when PV is deleted
+			err = volumeMigrationService.DeleteVolumeInfo(ctx, volumeHandle)
+			if err != nil {
+				log.Errorf("PVDeleted: failed to delete volumeInfo CR for volume: %q. Error: %+v", volumeHandle, err)
```

Comment:
Can you add a TODO here that says that we are not handling CR deletion failures now and this is tracked in https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/441?

Reply:
Done